### PR TITLE
Fix non-escaped hyphen in ContactProperties doc

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Ditto.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: docs/reference/contactproperties.md [hunk 1]
```diff
@@ -73,8 +73,7 @@ Solid objects with velocities below this threshold will have a `bounce` value se
 Consider a contact point where the coefficient of friction mu is infinite.
 Normally, if a force f is applied to the two contacting surfaces, to try and get them to slide past each other, they will not move.
 However, if the FDS coefficient is set to a positive value k then the surfaces will slide past each other, building up to a steady velocity of k*f relative to each other.
-Note that this is quite different from normal frictional effects: the force does not cause a constant acceleration of the surfaces relative to each other
-- it causes a brief acceleration to achieve the steady velocity."
+Note that this is quite different from normal frictional effects: the force does not cause a constant acceleration of the surfaces relative to each other \- it causes a brief acceleration to achieve the steady velocity."
```
Comment (omichel) [inline docs/reference/contactproperties.md]: Shouldn't use a mdash: &mdash; here?

Comment (omichel) [reply docs/reference/contactproperties.md]: (or a ndash: &ndash; ?)

Comment (stefaniapedrazzi) [reply docs/reference/contactproperties.md]: I would use the mdash.

Note that ODE manual really uses a hypen (even if it is "wrong").

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by omichel** [comment]
I would have typed "`&mdash;`" in the source code as I fear this UTF-8 copy/paste of a em dash sign will be replaced by a minus sign when we edit it later (depending on the font, it's difficult to distinguish it from a minus sign). Similarly we prefer writing "`&pi;`" rather than &pi; in the doc.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**